### ISSUE TITLE
Can we merge all duotone themes?

### ISSUE THREAD

**Issue by ghost** [simurai/duotone-syntax#10]
Just a question (not sure if this question is correctly placed in here).

Can we merge all themes by @simurai into one package with a settings page?

If it's wanted and done, sometimes later we can simplify this theme package to something like a duotone generator. Define two colors, auto generate shades, apply the new theme. And we can help offering great duotone color combinations. Duotone should be the de facto syntax theme standard.

**Comment by simurai** [comment]
Yeah, I tried that once here: https://github.com/simurai/duotone-dark-syntax/pull/4

In the settings you could pick a color variation. But it's kinda a hack and not really supported in Atom, so I closed it again. It would be pretty cool if you could just pick your own 2 colors. Maybe one day.

**Comment by ghost** [comment]
Cool!
It's a starter. I like it. Duotone is the best syntax theme!

**Comment by alehed** [comment]
I'm quite confused about the different variations of the theme: duotone (custom and fixed hues), tone and chameleon. Are any of those being deprecated?

Is it technically possible to have one theme (like chameleon) where one can either set custom hues or choose from a set of predefined ones?

**Comment by simurai** [comment]
@alehed Yes, eventually all the DuoTone (+chameleon) themes will be deprecated and replaced with:

- https://github.com/simurai/tone-dark-syntax
- https://github.com/simurai/tone-light-syntax

![screen shot 2017-04-18 at 9 19 34 am](https://cloud.githubusercontent.com/assets/378023/25109535/405d60b4-2418-11e7-9b35-e5310d49205c.png)

They're not published yet, and the name might still change. But you could already try them by entering the GitHub URL into Atom's `Settings > Install`.

**Comment by alehed** [comment]
@simurai Thanks! I'll check them out.

It would be nice to have presets available some day for those themes (I really like some of your color combinations), but for now I guess I can just keep the default values.
